scripts/keep_sync.py

```diff
@@ -99,7 +99,9 @@ def parse_raw_data_to_nametuple(run_data, old_gpx_ids, with_download_gpx=False):
             if str(keep_id) not in old_gpx_ids:
                 download_keep_gpx(gpx_data, str(keep_id))
         run_points_data = [[p["latitude"], p["longitude"]] for p in run_points_data]
-    heart_rate = run_data["heartRate"].get("averageHeartRate", None)
+    heart_rate = None    
+    if run_data["heartRate"] :   
+        heart_rate = run_data["heartRate"].get("averageHeartRate", None)
```

Comment:
```python
heart_rate = run_data.get("heartRate", {}).get("averageHeartRate", None)
```
may better ? seems can drop the `if`

Reply:
same error:
`Something wrong paring keep id 580c7a7831f1162b4c0041ea_9223370548826934807_rn'NoneType' object has no attribute 'get'`

trace back:

Traceback (most recent call last):
  File "scripts\keep_sync.py", line 233, in <module>
    run_keep_sync(options.email, options.password, options.with_gpx)
  File "scripts\keep_sync.py", line 169, in run_keep_sync
    new_tracks = get_all_keep_tracks(email, password, old_tracks_ids, with_download_gpx)
  File "scripts\keep_sync.py", line 151, in get_all_keep_tracks
    track = parse_raw_data_to_nametuple(

  File "scripts\keep_sync.py", line 102, in parse_raw_data_to_nametuple
    heart_rate = run_data.get("heartRate", {}).get("averageHeartRate", None)

AttributeError: 'NoneType' object has no attribute 'get'

Reply:
got it.

Reply:
Seems that  the 580c7a7831f1162b4c0041ea_9223370548826934807_rn has no runData, could help me to check.

Reply:
it has run_data, just no "HeartRate". with my ugly change I can import other information in run_data

Reply:
I think your way is right.